docs/sources/flow/monitoring/debugging.md

```diff
@@ -69,6 +69,17 @@ The component detail page shows the following information for each component:
 
 [secret]: {{< relref "../config-language/expressions/types_and_values.md#secrets" >}}
 
+### Clustering page
+
+![](../../../assets/ui_clustering_page.png)
+
+The clustering page shows the following information for each cluster node:
+
+* The node's name.
+* The node's advertised address.
+* The current state of the node (Viewer/Participant/Terminating).
```

Comment:
I think we should include a brief explanation for what each of these states means w/r/t clustering.

Reply:
I took a stab to add a high-level explanation in the debugging page, let me know if it reads better now.